packages/paste-website/src/components/component-overview-table/index.tsx

```diff
@@ -1,63 +1,105 @@
 import * as React from 'react';
 import {Box} from '@twilio-paste/box';
 import {Table, THead, TBody, Tr, Th, Td} from '@twilio-paste/table';
+import {Popover, PopoverContainer, PopoverButton} from '@twilio-paste/popover';
+import {Text} from '@twilio-paste/text';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {FigmaStatus} from '../component-status/FigmaStatus';
+import {PackageStatus} from '../component-status/PackageStatus';
+import {PeerReviewStatus} from '../component-status/PeerReviewStatus';
 import {SiteLink} from '../SiteLink';
-import {SidebarCategoryRoutes, PackageStatus} from '../../constants';
-import {getPackagePath, getHumanizedNameFromPackageName} from '../../utils/RouteUtils';
-import {ComponentNode} from '../../utils/types';
-import {filteredComponents, sortNodeByName} from '../../utils/componentFilters';
+import type {SidebarCategoryRoutes} from '../../constants';
+import {getPackagePath} from '../../utils/RouteUtils';
+import type {AirTableComponentNode} from '../../utils/types';
 
 interface ComponentOverviewTableProps {
   children?: React.ReactElement;
   categoryRoute?: typeof SidebarCategoryRoutes[keyof typeof SidebarCategoryRoutes];
-  componentsList?: [ComponentNode];
+  componentsList?: [AirTableComponentNode];
 }
 
 const ComponentOverviewTable: React.FC<ComponentOverviewTableProps> = ({categoryRoute, componentsList}) => {
   if (componentsList == null || categoryRoute == null) {
     return null;
   }
 
-  // Sort backlog items to the bottom of the list
-  const sortedBacklogList = componentsList
-    .filter(({node}) => node.status === PackageStatus.BACKLOG)
-    .sort(sortNodeByName);
-  const sortedComponentsList = componentsList.filter(filteredComponents).sort(sortNodeByName);
-
   return (
     <Box marginTop="space60" marginBottom="space60">
       <Table>
         <THead>
           <Tr>
             <Th>Name</Th>
-            <Th>Status</Th>
-            <Th>Version</Th>
+            <Th textAlign="center" width="150px">
+              Code ready
+            </Th>
+            <Th textAlign="center" width="150px">
+              Design assets
+            </Th>
+            <Th textAlign="center" width="150px">
+              Documentation
+            </Th>
+            <Th textAlign="center" width="150px">
+              <Box display="flex" alignItems="center">
+                <Text as="span" marginRight="space20">
+                  Peer review
+                </Text>
+                <PopoverContainer baseId="popover-example">
+                  <PopoverButton variant="link" size="icon_small">
+                    <InformationIcon decorative={false} title="More information about peer review" />
+                  </PopoverButton>
+                  <Popover aria-label="Popover">
```

Comment:
Ordinarily the purpose of labeling a dialog is to describe the dialog. Just in the same way we give modal headings, this is a none visual substitute to describe what the dialog is for.

Reply:
Thanks for catching. Copy/pasta miss.